emperor/support_files/js/emperor.js

```diff
@@ -887,17 +889,27 @@ function toggleTaxaLabels(){
 }
 
 /* Turn on and off the spheres representing the biplots on screen */
-function toggleBiplotVisibility(){
+function toggleBiplotVisibility(toggleArrow){
 	// reduce the opacity to zero if the element should be off or to 0.5
 	// if the element is supposed to be present; 0.5 is the default value
-	if(!document.biplotsvisibility.elements[0].checked){
+        toggleArrow = typeof toggleArrow !== 'undefined' ? toggleArrow : false
+        arrowBox = toggleArrow ? 1 : 0
+	if(!document.biplotsvisibility.elements[arrowBox].checked){
```

Comment:
I think all this can be collapsed quite a bit more. My js is terrible, so I apologize if this is a bit off:

```javascript
updater = document.biplotsvisibility.elements[arrowBox].checked ? function(data, index) {g_mainScene.add(data[index])} : function(data, index) {g_mainScene.remove(data[index])};
data_to_update = toggleArrow ? g_plotTaxaArrows : g_plotTaxa;

for (index in g_plotTaxa) {
    updater(data_to_update, index);
}
```

It has the advantage of reducing the LOC and pulling out branching logic from within the forloop at the expense of some ugly lines.